proposals/0091-SDLScreen-SDLWindow-Projection.md

```diff
@@ -1,110 +1,209 @@
+The iOS SDL framework implements an SDLCarWindow class to provide an easy-to-integrate developer interface to SDL video projection the associated task of defining focusable and selectable UI elements and the translation of remote touch events into standard UIKit actions. 
 
 ## Motivation
 
-This proposal builds on the SDLStreamingMediaManager to provide a convenient and simple programming model for SDL app developers to project and enable interaction with the appropriate subset of their UIKit, OpenGL and hybrid UIKit/OpenGL user interfaces.
+Provide a convenient and simple programming model for SDL app developers to remotely project and enable interaction with the appropriate subset of their UIKit, OpenGL and hybrid UIKit/OpenGL user interfaces. [1]
 
 ## Proposed solution
 
-The SDLStreamingMediaManager class is extended to include a car window property:
-
-```objc  
-@interface SDLStreamingMediaManager : NSObject <SDLProtocolListener>
-:
-:
-/**
- *  Touch Manager responsible for providing touch event notifications.
- */
-@property (nonatomic, strong, readonly) SDLTouchManager *touchManager;
-
-/**
-* Car Window instance responsible for providing video projection.
-* /
-@property (nonatomic, strong, readonly) SDLCarWindow *carWindow;
-:
-:
-```
-The SDLCarWindow interface is defined as follows:
+SDLCarWindow leverages SDLStreamingMediaManager and SDLTouchManager to hide the management of video projection and touch event handling from the app developer. For apps which use only UIKit derived UI elements, no SDL specific code is necessary. [1]
 
-```objc
-@interface SDLCarWindow : NSObject
+SDLTouchManager is embellished to invoke view actions based on remote touch events.
+
+If the head unit communicates that it implements a focus/select interaction model rather than direct touch, the focusableItemLocatorEnabled property will be set to YES. 
+
+**Application Interface**
 
-/**
- *  View Controller that will be streamed.
- */
-@property (strong, nonatomic, nullable) UIViewController *streamingViewController;
+The video projection interface is exposed to the application as a UIScreen object representing a virtual external display. The external view is separate from the view rendered on the device's LCD. This allows the developer to optionally forego the lock screen, instead displaying a different interface on the device than what is projected onto the head unit.
 
-/**
- * Dimensions of the target in-car screen
- */
-@property (assign, nonatomic, readonly) CGSize screenDimensions;
+![Separate Displays](https://i.imgur.com/dIBQnyZ.png "Separate Displays")
```

Comment:
Could you please upload this image to the /assets/proposals folder and link to that image?  You can reference [SDL 0084](https://github.com/smartdevicelink/sdl_evolution/issues/250) as an example.

Reply:
Done. Will I need a separate PR for this graphic?